python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_idf.yaml

```diff
@@ -1,13 +1,17 @@
-pipelineYaml: "${OBS_SUBARU_DIR}/pipelines/DRP.yaml#processCcd"
+includeConfigs:
+- ${CTRL_BPS_DIR}/config/bps_idf.yaml
+
+operator: ${USER}
+project: dev
+campaign: quick
+pipelineYaml: "${OBS_LSST_DIR}/pipelines/imsim/DRP.yaml#step1"
 
 payload:
   payloadName: pipelines_check
-  butlerConfig: s3://butler-us-central1-panda-dev/hsc/butler.yaml
-  inCollection: HSC/calib,HSC/raw/all,refcats
-  dataQuery: "tract = 9615 and patch=30 and detector IN (10..11) and instrument='HSC' and skymap='hsc_rings_v1' and band in ('r')"
-  output: u/{operator}/pcheck_ebutler
+  output: "u/{operator}/{payloadName}"
+  butlerConfig: s3://butler-us-central1-panda-dev/dc2/butler.yaml
+  inCollection: "2.2i/defaults/test-med-1"
+  dataQuery: "instrument='LSSTCam-imSim' and skymap='DC2' and exposure in (214433) and detector<5"
 
   sw_image: "lsstsqre/centos:7-stack-lsst_distrib-w_2021_39"
-  fileDistributionEndPoint: "s3://butler-us-central1-panda-dev/hsc/{payload_folder}/{uniqProcName}/"
-  s3_endpoint_url: "https://storage.googleapis.com"
-  payload_folder: payload
+  fileDistributionEndPoint: "s3://butler-us-central1-panda-dev/dc2/{payload_folder}/{uniqProcName}/"
```

Comment:
Is this something regular users tend to change?  If not it can go into the bps_idf.yaml.

Reply:
Most people won't change it but I hesitate if it should match the butler repo.  I moved it to bps_idf.yaml.